update-bootengine

```diff
@@ -104,6 +104,13 @@ else
     DRACUT_ARGS+=( "--no-kernel" )
 fi
 
+if [ -f /.dockerenv ] ; then
```

Comment:
I think we can even make this unconditional to have a consistent behavior

Reply:
We are lucky it hit a test case in kola, I think it's best to have the same behavior regardless of old SDK or new SDK and whether it's building in / or /build/BOARD.